tests/functional/acp_attachments_test.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+/**
+ * @group functional
+ */
+class phpbb_functional_acp_attachments_test extends phpbb_functional_test_case
+{
+	public function data_imagick_path()
+	{
+		return array(
+			array('/usr/bin', 'Configuration updated successfully'),
+			array('/usr/bin/', 'Configuration updated successfully'),
+			array('C:\Windows\system32', 'The entered path “C:\Windows\system32” does not exist.'),
+			array('/usr/nope', 'The entered path “/usr/nope” does not exist.'),
+			array('mkdir /usr/test', 'The entered path “mkdir /usr/test” does not exist.'),
+			array('/usr/bin/which', 'The entered path “/usr/bin/which” is not a directory.'),
```

Comment:
What about a test case for `.` which should fail, but doesn't look like it will.